Tests/LibCore/TestLibCoreStream.cpp

```diff
@@ -300,19 +300,25 @@ TEST_CASE(local_socket_read)
     auto local_server = Core::LocalServer::construct();
     EXPECT(local_server->listen("/tmp/test-socket"));
 
+    Atomic<unsigned> should_exit_condition { 0 };
+
     local_server->on_accept = [&](NonnullRefPtr<Core::LocalSocket> server_socket) {
         EXPECT(server_socket->write(sent_data));
 
+        while (should_exit_condition.load(AK::memory_order_acquire) != 0xDEADBEEF) {
+            // Let's snooze a little until the condition becomes what we expect.
+            usleep(10000);
```

Comment:
I guess this works and fixes the CI, but the whole commit feels a little hackish to me.

Reply:
Dropped this commit in favor of making the `local_socket_*` tests less thread-y in another PR.